### DISCUSSION TITLE
Reward did not come

### DISCUSSION THREAD

**Discussion by chgs1** [Chia-Network/chia-blockchain#11579] [Support]
Good afternoon!

I solved the block, but the full reward did not come.
Only 0.25 XCH was received and 1.75 XCH was not received.
Grown Block Height: 1999288

The address:
xch1gh8q0w9tv3p3vypz7wgrsql4k533mf9un3kq80yncgt3kc90ur9qa3zlsu

I had a similar situation on December 20, 2021 4:29 am, only 0.25 XCH was also received, and 1.75 XCH has not yet come.
Grown Block Height: 1303122

Please help me understand the situation.

Thank you!

**Comment by FuzeGuy** [comment]
Did you find it? Are you farming NFT plots (self-pooling)? If so, only .25 comes to the main address, the 1.75 comes in on the pool address, and must be 'claimed' on the Pooling tab in the GUI. Then the total 2 XCH will be shown in your wallet.

**Reply by chgs1** [reply]
Any one issue or fix for this?...

Farming only OG-plots.
The block was found by me

**Reply by FuzeGuy** [reply]
If u have only non-NFT plots (OG) then 2 xch would come to the main address. A thought, have u ever pooled? If so that may be part of the answer, there's some remnant of the old pool addresss in ur setup. Otherwize, something is wrong and/or different that u describe as it is continuing to occur over time. It has to be u.

Can u install new on another pc, enter your passphrase, and see if the coin comes back?

**Reply by FuzeGuy** [reply]
Does this show ur correct XCH?

https://blockchain.chiaexplorer.com/blockchain/address/xch1gh8q0w9tv3p3vypz7wgrsql4k533mf9un3kq80yncgt3kc90ur9qa3zlsu

**Reply by chgs1** [reply]
> Does this show ur correct XCH?
> 
> https://blockchain.chiaexplorer.com/blockchain/address/xch1gh8q0w9tv3p3vypz7wgrsql4k533mf9un3kq80yncgt3kc90ur9qa3zlsu

Yes, this show correct.

The transactions also show that only 0.25XCH was received (Block Height: 1999288 and Block Height: 1303122)

**Reply by chgs1** [reply]
> If u have only non-NFT plots (OG) then 2 xch would come to the main address. A thought, have u ever pooled? If so that may be part of the answer, there's some remnant of the old pool addresss in ur setup. Otherwize, something is wrong and/or different that u describe as it is continuing to occur over time. It has to be u.
> 
> Can u install new on another pc, enter your passphrase, and see if the coin comes back?

The settings are correct. They didn't change during the year. The rest of the production (except for two times) comes in full.

**Reply by FuzeGuy** [reply]
I've got nothing then. perhaps post iover on Keybase and see if they can come up with any reasonable explanation. Good luck, it is very strange.

**Comment by chgs1** [comment]
Hello!

Thanks for the answer!
Yes, the block was found by me.
Farming only OG-plots.

пн, 23 мая 2022 г. в 04:01, FuzeGuy ***@***.***>:

**Comment by 1-justinstarr** [comment]
Sorry for delayed response.. I believe I tried to pool but the process was interrupted. I have had net issues since

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
